feature/interface/singleton/ate_tests/singleton_test/singleton_test.go

```diff
@@ -238,7 +238,7 @@ func (tc *testCase) verifyInterfaceDUT(
 			}
 		}
 	}
-	if *deviations.MissingSubinterfaceOriginLeaf {
+	if !*deviations.SubinterfaceOriginLeaf {
```

Comment:
If we are only skipping the origin, we should still get something from the neighbor to ensure that the neighbor exists.